### PR TITLE
Unifying indentation: more-python-for-beginners/chapter 1-3 

### LINKED ISSUE #26: Unifying indentation: more-python-for-beginners chapter 1-3
This is sub-issue derived from #20

### PR DESCRIPTION
**Pull request by Dorika0203**
I've fixed indent for more-python-for-beginners/chap 1~3, issue of #26 
Also, I've wrote some additional examples in code.

### CODE REVIEW

**Review by devquint — CHANGES_REQUESTED** [review]
Do not change any of the code here, use another branch not under `indent-unify`.

**Comment by Dorika0203** [comment]
Oh, okay

**Comment by Dorika0203** [comment]
I've reset the commits before and added new commits. @devquint

File: more-python-for-beginners/01 - Formatting and linting/bad.py [hunk 1]
```diff
@@ -1,11 +1,11 @@
 x = 12
 if x == 24:
- print('Is valid')
+    print('Is valid')
 else:
     print("Not valid")
 
 def helper(name='sample'):
- pass
+    pass
```
Comment (devquint) [inline more-python-for-beginners/01 - Formatting and linting/bad.py]: I think `bad.py` should be indented badly, as original.

File: more-python-for-beginners/02 - Lambdas/failed_sort.py [hunk 2]
```diff
@@ -1,9 +1,9 @@
 # This code will return an error because the sort method does not know
 # which presenter field to use when sorting
 presenters = [
-    {'name': 'Susan', 'age': 50},
-    {'name': 'Christopher', 'age': 47}
-]
+        {'name':'Susan', 'age':50},
```
Comment (devquint) [inline more-python-for-beginners/02 - Lambdas/failed_sort.py]: Should be 1 level(4 spaces).

File: more-python-for-beginners/02 - Lambdas/failed_sort.py [hunk 3]
```diff
@@ -1,9 +1,9 @@
 # This code will return an error because the sort method does not know
 # which presenter field to use when sorting
 presenters = [
-    {'name': 'Susan', 'age': 50},
-    {'name': 'Christopher', 'age': 47}
-]
+        {'name':'Susan', 'age':50},
+        {'name':'Christopher', 'age':47}
```
Comment (devquint) [inline more-python-for-beginners/02 - Lambdas/failed_sort.py]: Should be 1 level(4 spaces).

File: more-python-for-beginners/02 - Lambdas/failed_sort.py [hunk 4]
```diff
@@ -1,9 +1,9 @@
 # This code will return an error because the sort method does not know
 # which presenter field to use when sorting
 presenters = [
-    {'name': 'Susan', 'age': 50},
-    {'name': 'Christopher', 'age': 47}
-]
+        {'name':'Susan', 'age':50},
+        {'name':'Christopher', 'age':47}
+        ]
```
Comment (devquint) [inline more-python-for-beginners/02 - Lambdas/failed_sort.py]: Should be 0 level(0 spaces).

File: more-python-for-beginners/02 - Lambdas/lambda_sorter.py [hunk 5]
```diff
@@ -1,14 +1,14 @@
 # Sort alphabetically
 presenters = [
-    {'name': 'Susan', 'age': 50},
-    {'name': 'Christopher', 'age': 47}
-]
+        {'name':'Susan', 'age':50},
+        {'name':'Christopher', 'age':47}
+        ]
```
Comment (devquint) [inline more-python-for-beginners/02 - Lambdas/lambda_sorter.py]: Same as above(`filed_sort.py`)

File: more-python-for-beginners/02 - Lambdas/method_sorter.py [hunk 6]
```diff
@@ -3,8 +3,8 @@ def sorter(item):
 
 
 presenters = [
-    {'name': 'Susan', 'age': 50},
-    {'name': 'Christopher', 'age': 47}
-]
+        {'name':'Susan', 'age':50},
+        {'name':'Christopher', 'age':47}
+        ]
```
Comment (devquint) [inline more-python-for-beginners/02 - Lambdas/method_sorter.py]: Same as above(`filed_sort.py`)

**Review by devquint — CHANGES_REQUESTED** [review]
There's no other problem than I commented each file/line, check them please.

**Comment by Dorika0203** [comment]
@devquint Sorry about too much wrong indenting. I've found out the reason - using vim editor indenting caused this level problem. Thanks for your review.

**Review by devquint — APPROVED** [review]

**Review by Hyuntae-Jeong — APPROVED** [review]

**Review by jesusleejy — APPROVED** [review]

**Pull request merged**